Documentation/README_Solver.md

```diff
@@ -22,6 +22,8 @@ The third category is the solver itself. The following solvers provide the build
 * **SurfaceMagnetism**, casts rays to surfaces in the world, and align the object to that surface.
 * **Momentum**: Applies acceleration/velocity/friction to simulate momentum and springiness for an object being moved by other solvers/components.
 * **InBetween**: Keeps an object in between two tracked objects.
+* **HandConstraint**: Keeps an object in a region safe for hand constrained interactive content.
```

Comment:
Contrains object to follow hands in a region that doesn't intersect the GameObject with the hands. Useful for hand constrained interactive content such as menus etc.

region safe is a big question mark until I read the section below